### ISSUE TITLE
All interactive elements are actionable when navigating without a mouse.

### ISSUE THREAD

**Issue by justpie** [Purdue/purdue-wp-theme#123]
When keyboard and screen-reader users press the "Close(X)" button in the expanded image modal, the modal cannot be closed since it is implemented using a <div> with an aria-label, which is not a keyboard-focusable or interactive element. Use a semantic <button> element for the close button so it is keyboard accessible and properly conveyed to screen-reader users.

**Comment by justpie** [comment]
This happens across a number of blocks. Due to how the javascript is written, I needed to update all the blocks. Additionally, the blocks were missing a focus trap, and i made some additional efforts to support iframes (youtube embeds) with the focustrap as well.
